apps/ui/lib/components/InfiniteList.tsx

```diff
@@ -15,7 +15,9 @@ const ScrollArea = styled<any>(Box)`
 	overflow-y: auto;
 	display: flex;
 	flex-direction: column;
+	padding-bottom: 100px;
```

Comment:
The additional bottom padding works fine here - its only an issue when you're at the most recent message